pkg/gardenadm/staticpod/translator.go

```diff
@@ -36,7 +35,7 @@ func Translate(ctx context.Context, c client.Client, o client.Object) ([]extensi
 func translatePodTemplate(ctx context.Context, c client.Client, objectMeta metav1.ObjectMeta, podTemplate corev1.PodTemplateSpec) ([]extensionsv1alpha1.File, error) {
 	pod := &corev1.Pod{ObjectMeta: podTemplate.ObjectMeta, Spec: podTemplate.Spec}
 	pod.Name = objectMeta.Name
-	pod.Namespace = objectMeta.Namespace
+	pod.Namespace = metav1.NamespaceSystem
```

Comment:
We use `pod.Namespace` for looking up referenced secrets and configmaps. If the input is in another namespace than `kube-system` this would fail now, right?